src/index.html

```diff
@@ -16,7 +20,351 @@
       defer
     ></script>
   </head>
-  <body>
-    <h1>Landing Page</h1>
+  <!-- #endregion -->
+  <body class="page__body">   
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-bar">
+            <a href="#" class="top-bar__logo">
+              <p class="top-bar__logo-text">
+                THE
+                <br
+                >MET
+              </p>
+            </a>
+            <div class="top-bar__icons">
+              <a href="#" class="icon icon--phone"></a>
+              <a href="#menu" class="icon icon--menu"></a>
+            </div>
+          </div>
+        
+          <div class="header__center">
+            <h1 class="header__title">
+              Welcome
+              <br>
+              to The MET
+            </h1>
+          </div>
+        </div>
+      </div>
+    </header>
+    
+    <aside class="page__menu menu" id="menu">
+      <div class="container">
+        <div class="top-bar menu__top">
+          <a href="#" class="top-bar__logo">
+            <p class="top-bar__logo-text">
+              THE
+              <br
+              >MET
+            </p>
+          </a>
+          <div class="header__icons">
+            <a href="#" class="icon icon--close"></a>
+          </div>
+        </div>
+        
+        <div class="menu__bottom">
+          <nav class="nav menu__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link" href="#home">Home</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#about-us">About us</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#now-on-view">Now on view</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#explore-online">Explore online</a>
+              </li>
+
+              <li class="nav__item">
+                <a class="nav__link" href="#contacts">Contacts</a>
+              </li>
+            </ul>
+          </nav>
+          <p class="menu__text">The met firth avenue</p>
+          <a href="tel:+1 212-535-7710" class="menu__phone-number">
+            +1 212-535-7710
+          </a>
+          <p class="menu__text">The met cloisters</p>
+          <a href="tel:+1 212-923-3700" class="menu__phone-number">
+            +1 212-923-3700
+          </a>
+          <a href="tel:+1 212-923-3700" class="menu__call-to-order">
+            Call to order
+          </a>
+        </div>
+      </div>
+    </aside>
+
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <section class="about-us" id="about-us">
+            <div class="about-us__container-text">
+              <p class="
+              section-title--up
+              section-title--align--left
+              about-us__container-text__title"
```

Comment:
I don't like that the name of each class starts on a new line
try shortening the length of the classes or:
```
<p
  class="section-title--up section-title--align--left about-us__container-text__title"
>
```

Reply:
length of row is 86 symbols